.babelrc

```diff
@@ -0,0 +1,22 @@
+{
+  "presets": ["react", "es2015"],
```

Comment:
I do not have professional experience with React or ECMAScript 2015. I'm studying it for personal projects and when this challenge shows up, I chose it, because I was charmed by the concepts (Virtual DOM, functional programming) and I thought it would be cool to put into practice what I am learning.